_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -241,6 +243,76 @@ mfxStatus FallbackProxyENCODE::Init(mfxVideoParam *par)
 }
 
 
+struct Handlers {
+    typedef mfxStatus (*QueryFunc)(mfxSession s, mfxVideoParam *in, mfxVideoParam *out);
```

Comment:
`decltype(MFXVideoENCODE_Query)`? @mgonchar : how do you think?

Reply:
Yes, I thought about removing typedef. But indeed for that we should not have overloads. Anyway, that's minor. Let's form the solution first, then decide about this place.